src/core/Jalasoft.TeamUp.Resumes.Core.Interfaces/IHealthService.cs

```diff
@@ -5,5 +5,7 @@
     public interface IHealthService
     {
         Health GetHealth();
+
+        Health PostHealth();
```

Comment:
it's not necesarry add a posthealth function

Reply:
Health PostHealth() was remove